api/rpc/logs/logs.go

```diff
@@ -72,7 +71,7 @@ const (
 type Server struct {
 	Es            *elasticsearch.Elasticsearch
 	Store         storage.Interface
-	NatsStreaming ns.NatsStreaming
+	MessageQueuer mq.Interface
```

Comment:
No offense, but `MessageQueuer` is pretty unwieldy ;).  I would just call it `MQ` and if you want to make things consistent, you can rename `Store` here to `KV`.